builders/xcm/xcm-sdk/xcm-sdk.md

```diff
@@ -0,0 +1,849 @@
+|       `native`       |                                                                                                                                                  the native [asset](#assets) of the source chain                                                                                                                                                   |
+|       `origin`       |                                                                                                                                the chain information for where the asset being transferred natively originates from                                                                                                                                |
+|       `source`       |                                                                                                                                      the chain information for where the asset being transferred is sent from                                                                                                                                      |
+|   `sourceBalance`    |                                                                                                                                           the balance of the asset being transferred on the source chain                                                                                                                                           |
+|  `sourceFeeBalance`  |                                                                                                                                            the fee for the asset to be transferred on the source chain                                                                                                                                             |
+|  `sourceMinBalance`  |                                                                                                                                       the minimum balance of the asset being transferred on the source chain                                                                                                                                       |
+|       `getFee`       |                                                                                                                               a function that [estimates the fees](#get-fee-withdraw) for depositing a given amount                                                                                                                                |
+|        `send`        |                                                                                                                                            a function that [sends](#send-withdraw) the withdraw request                                                                                                                                            |
```

Comment:
@ekenigs Can you please review these definitions? I'm not so sure they're correct haha started guessing 😅